packages/backend-core/src/db/db.ts

```diff
@@ -7,11 +7,11 @@ const dbList = new Set()
 
 export function getDB(dbName?: string, opts?: any): Database {
   // TODO: once using the test image, need to remove this
-  if (env.isTest()) {
-    dbList.add(dbName)
-    // @ts-ignore
-    return getPouchDB(dbName, opts)
-  }
+  // if (env.isTest()) {
+  //   dbList.add(dbName)
+  //   // @ts-ignore
+  //   return getPouchDB(dbName, opts)
+  // }
```

Comment:
Should this be removed now?

Reply:
Still a WIP - should have marked as draft!